tests/renderer/setuptab.test.js

```diff
@@ -3,6 +3,7 @@ import React from 'react';
 import {
   createEvent, fireEvent, render, waitFor, within

```

Comment:
Are there remaining uses of `fireEvent` that could be changed to use `userEvent`?

Reply:
Yeah, the remaining uses are for the drag-and-drop tests. There was no clear way to trigger those events with `userEvent`. Here's a convo about it: https://github.com/testing-library/user-event/issues/440